src/Test/SolidifyProject.Engine.Test.Integration/Infrastructure/Services/OrchestrationServiceTest.cs

```diff
@@ -42,11 +42,67 @@ public class OrchestrationServiceTest
 
         private IHtmlMinificationService HtmlMinificationService => new GoogleHtmlMinificationService();
 
+        [SetUp]
+        public void CleanUpBeforeTest()
+        {
+            _assetsWriterStorage.Clear();
+            _pageModelWriterStorage.Clear();
+        }
+
         [Test]
         public async Task ProduceHtml()
         {
-            Assert.AreEqual(0, _pageModelWriterStorage.Count);
-            Assert.AreEqual(0, _assetsWriterStorage.Count);
```

Comment:
why it was removed?

Reply:
Because it's can be not empty fields are shared between tests

Reply:
then we need to create a `[TestFixtureSetUp]` and `[TestFixtureTearDown]` methods to make sure that folder is clean before and after test runs